Rubberduck.Core/Common/CodeString.cs

```diff
@@ -1,12 +1,44 @@
 ﻿using Rubberduck.VBEditor;
 using System;
+using System.Diagnostics;
 
 namespace Rubberduck.Common
 {
+    public class TestCodeString : CodeString
+    {
+        public static readonly char PseudoCaret = '|';
```

Comment:
We want to avoid making it impossible to write unit-tests with code that contains `|`, which is a legitimate character. Replacing with something that the VBE doesn't handle would make sense regardless of whether this class interacts with the VBE

Reply:
But writing unit tests that use `|` to identify caret position is intuitive, and doing so only makes the legit `|` character awkward in tests that need a `TestCodeString` - pipe char has no meaning in VBA, it's only legal in a string literal. Does writing tests that contain legit `|` characters outweight the simplicity and intuitiveness of using it for tests that need a caret position? I don't mind changing the `PseudoCaret` character, but then I feel it would only make it harder to write the tests that *do* need it.